code/mob/living/silicon/robot.dm

```diff
@@ -2317,7 +2316,7 @@
 
 	proc/borg_death_alert(modifier = ROBOT_DEATH_MOD_NONE)
 		var/message = null
-		var/mailgroup = MGD_MEDRESEACH
+		var/list/mailgroups = list(MGD_MEDRESEACH, "silicon")
```

Comment:
We should probably use constants for new mailgroups (i.e. make one for `"silicon"`), I think?